### PR TITLE
Cleanup project badges

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -22,13 +19,6 @@ Jira Python Library
 .. image:: https://readthedocs.org/projects/jira/badge/?version=master
     :target: https://jira.readthedocs.io/
 
-.. image:: https://travis-ci.com/pycontribs/jira.svg?branch=master
-    :target: https://travis-ci.com/pycontribs/jira
```
Comment (adehad) [inline README.rst:26]: Might be able to at the GHA badge here:

https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge

**Review by studioj — APPROVED** [review]
Just want to make sure that removing black is not by accident

File: README.rst (hunk 1 above)
Comment (ssbarnea) [reply README.rst:26]: I considered it but decided to delay until we are sure it is stable ;)

**Comment by ssbarnea** [comment]
> Just want to make sure that removing black is not by accident



> Just want to make sure that removing black is not by accident

I love black, but I think it needs no more advertising. Its use is not relevant to the project.

**Review by adehad — APPROVED** [review]

**Pull request merged**